source/pages/epoch.scss

```diff
@@ -0,0 +1,52 @@
+.epochLayout {
+  header {
+    button {
+      background: transparent !important;
+      border: 1px solid #eb2256;
+
+      > div {
+        border-left: 9px solid #eb2256 !important;
+      }
+    }
+  }
+
+  .container {
```

Comment:
@DeeJayElly  You can remove `.container` style if you reuse `Container` widget component.

Reply:
Fixed